docker-compose.yml

```diff
@@ -34,8 +34,8 @@ services:
     command:
       - -f=/app
       - --db-schema=app
-      - --log-level=debug
-      - --disable-historical=false
+      - --log-level=error
+      - --disable-historical=true
```

Comment:
@Douglasacost did you find any issues with this option?

Reply:
This is only affecting local testing. The docker would never be deployed at least we take over our own deployments and infra